ethereum/core/src/test/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulatorTest.java

```diff
@@ -559,13 +560,14 @@ public void shouldCapGasLimitWhenRpcGasCapIsDefined() {
             OperationTracer.NO_TRACING,
             1L);
 
-    assertThat(result.get().isSuccessful()).isTrue();
+    assertThat(result.get().isSuccessful()).isFalse();
     verifyTransactionWasProcessed(expectedTransaction);
   }
 
   @Test
-  public void shouldKeepOriginalGasLimitWhenCapIsHigherThanOriginalValue() {
-    final CallParameter callParameter = eip1559TransactionCallParameter(Wei.ZERO, Wei.ZERO, 200L);
+  public void shouldApplyGasCapWhenOriginalGasLimitIsLowerThanGasCap() {
```

Comment:
Why do we apply the gas cap when gas limit is already lower?

Reply:
whoops, thanks for catching this.